app/vote_ext_utils.go

```diff
@@ -700,3 +702,7 @@ func packExtensionWithVote(extension []byte) []byte {
 	buf.Write(extension)
 	return buf.Bytes()
 }
+
+func shouldCheckVotingPower(currentHeight int64) bool {
+	return currentHeight < helper.GetDisableVPCheckHeight() || currentHeight >= helper.GetTallyFixHeight()
```

Comment:
Should it be `helper.GetTallyFixHeight() + 1`?  (not sure, just asking)

Reply:
Well `GetTallyFixHeight()` is the height at which we re-activate the checks regarding voting power and validator set. Why would you think it should be `GetTallyFixHeight() + 1` ?